app/controllers/vacation_requests_controller.rb

```diff
@@ -39,6 +52,14 @@ def update
     end
   end
 
+  def cancel
+    authorize @vacation_request
+    @vacation_request.update!(status: VacationRequest.statuses[:cancelled])
+    @vacation_request.approval_requests.destroy_all
```

Comment:
Maybe destroy `@vacation_request`? And approval_requests destroy dependently?
We don't need this vacation_request anymore. Right?